### PR TITLE
Add header

### PR DESCRIPTION
**Pull request by emphay**
nav-desktop
nav-mobile

### CODE REVIEW

**Review by EjazAhmd — CHANGES_REQUESTED** [review]
Remove extra paddings for mobile view not  as per design

**Review by EjazAhmd — CHANGES_REQUESTED** [review]
Add Screenshot

**Comment by emphay** [comment]
nav-mobile

File: blogs/01/styles/header.css [hunk 1]
```diff
@@ -1,89 +1,65 @@
...
     font-size: 24px;
-    color: #000 !important;
+    color: #000;
     font-weight: 400;
     padding: 0px 20px;
 }
-div.slide>ul{
+.menu
+{
     list-style-type: none;
```
Comment (amezng) [inline blogs/01/styles/header.css:18]: list-style-type should be a generic rule

File: blogs/01/styles/header.css [hunk 2]
```diff
@@ -1,89 +1,65 @@
...
     display: flex;
     justify-content: space-between;
     align-items: center;
     gap: 30px;
     padding: 0px 20px;
-    margin-top: -20px;
+    margin-top: 21px;
```
Comment (amezng) [inline blogs/01/styles/header.css:24]: don't use such values. Use 20, 25, 10. In multiples of 5 preferably

File: blogs/01/styles/header.css [hunk 3]
```diff
@@ -1,89 +1,65 @@
...
 }
-div.slide>ul>li>a{
+.menu > li > a
+{
     text-decoration: none;
     padding: 10px 0px;
     font-weight: 500;
     color: #000;
```
Comment (amezng) [inline blogs/01/styles/header.css:31]: Use variables for the colors

**Review by amezng — CHANGES_REQUESTED** [review]

File: blogs/01/styles/header.css (hunk 1 above)
Comment (amezng) [reply blogs/01/styles/header.css:18]: it should be in common css

**Review by EjazAhmd — APPROVED** [review]
LGTM

**Review by amezng — APPROVED** [review]
LGTM

**Pull request merged**